platform/stm32f0xx/scripts/pretty_size.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+size_data_raw=$($1 $2)
+echo "$size_data_raw"
+
+read -ra temp <<< "$(echo -e "$size_data_raw" | sed -n '2p')"
+
+data="${temp[1]}"
+bss="${temp[2]}"
+
+bytes_used=$(("$bss + $data"))
+bytes_total=16384 #16kb available
+bytes_remaining=$(("$bytes_total - $bytes_used"))
+bytes_threshold=2048 #need 2kb static memory left
+
+echo "Power distribution used ${bytes_used}/${bytes_total} bytes of static memory," \
+     "leaving $bytes_remaining bytes for stack space."
```

Comment:
Power distribution is just one project of many - instead of always saying "Power distribution", you should get the project from the arm-none-eabi-size output as the name of the .elf file without the .elf suffix. For example, if you build `smoke_i2c` with `make build PROJECT=smoke_i2c`, it should say `smoke_i2c used 184/16384 bytes of static memory...`